db/migrate/20121118120028_migrate_hypervisors_to_compute_resources.rb

```diff
@@ -1,18 +1,6 @@
 class MigrateHypervisorsToComputeResources < ActiveRecord::Migration[4.2]
-  class Hypervisor < ApplicationRecord; end
-
   def up
-    return unless Hypervisor.table_exists?
-
-    Hypervisor.all.each do |hypervisor|
-      # check if we have the same compute resource already, if we do, skip it.
-      next if Foreman::Model::Libvirt.find_by_url hypervisor.uri
-
-      Foreman::Model::Libvirt.create :name        => hypervisor.name,
-                                     :url         => hypervisor.uri,
-                                     :description => "Automatically migrated from hypervisor #{hypervisor.name} / #{hypervisor.uri}"
-    end
-    drop_table :hypervisors
+    drop_table :hypervisors, if_exists: true
   end
 
   def down
```

Comment:
It's weird to see an `up` migration that's a `drop_table` but an empty `down`.

Reply:
That's true, although it was that way before as well so no idea what should be in that table. Also, I highly doubt anyone would rollback migrations all they way to 2012 😜